assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -118,20 +127,22 @@ const CloseButton = styled(Button)`
 const SnackbarMessage = ({
   actionLabel,
   handleAction = () => {},
-  handleDismiss,
+  handleDismiss = () => {},
```

Comment:
💭 small nit: maybe this is a good chance to use our `noop` helper function since it won't be re-created on every render